### PR TITLE
Add doc comments

### PR DESCRIPTION
**Pull request by zalun**

### CODE REVIEW

File: fetching/index.js [hunk 1]
```diff
@@ -1,18 +1,25 @@
 // [Working example](/serviceworker-cookbook/fetching/).
 
+// These urls will be loaded several times
```
Comment (marco-c) [inline fetching/index.js]: Nit: URLs

File: push-subscription-management/index.js [hunk 2]
```diff
@@ -1,5 +1,8 @@
+// [Working example](/serviceworker-cookbook/push-subscription-management/).
+
 var subscriptionButton = document.getElementById('subscriptionButton');
 
+// Provide a Promise with subscription to not repeat too often
```
Comment (marco-c) [inline push-subscription-management/index.js]: Could you rephrase? I'm not sure what you meant here

File: push-subscription-management/server.js [hunk 3]
```diff
@@ -15,19 +19,21 @@ if (!process.env.GCM_API_KEY) {
   webPush.setGCMAPIKey(process.env.GCM_API_KEY);
 }
 
+// Send notification to the push service. Remove the endpoint from the
+// `subscriptions` array if push services responds with an error.
```
Comment (marco-c) [inline push-subscription-management/server.js]: if the push service

File: push-subscription-management/server.js [hunk 4]
```diff
@@ -1,13 +1,17 @@
+// [Working example](/serviceworker-cookbook/push-subscription-management/).
+
+// `web-push` is a library which makes sending notifications a very
+// simple process.
 var webPush = require('web-push');
-// collection of endpoints stored in memory (in production it should be
-// a database)
+// Global array collecting all active endpoints. In real live example
```
Comment (marco-c) [inline push-subscription-management/server.js]: Maybe "In a real world application" would be better

File: push-subscription-management/server.js [hunk 5]
```diff
@@ -15,19 +19,21 @@ if (!process.env.GCM_API_KEY) {
...
-    // there has been an error - it is possible the subscription has
-    // been cancelled or expired - remove the endpoint
     console.log('ERROR in sending Notification, endpoint removed ' + endpoint);
     subscriptions.splice(subscriptions.indexOf(endpoint), 1);
   });
 }
 
-// To simulate a real life notification server we're sending
-// a notification every [pushInterval] seconds to each endpoints
+// In real live notification is send only if an event occured.
```
Comment (marco-c) [inline push-subscription-management/server.js]: Here too

Comment (marco-c) [inline push-subscription-management/server.js]: is sent

File: push-subscription-management/server.js [hunk 6]
```diff
@@ -15,19 +19,21 @@ if (!process.env.GCM_API_KEY) {
...
+// To simulate it server is sending a notification every `pushInterval` seconds
+// to each registered endpoint.
```
Comment (marco-c) [inline push-subscription-management/server.js]: To simulate it, the server is

File: push-subscription-management/index.js (hunk 2 above)
Comment (zalun) [inline push-subscription-management/index.js]: As `subscription` object is needed in few places let's create a method which returns a promise.

Comment (marco-c) [inline push-subscription-management/index.js]: Ah, ok! I'd rephrase the comment to make it clearer
